samples/c_cxx/opschema_lib_use/dummy.cc

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <string>
+#include <vector>
+
+// Dummy implementations of dependencies used by op schema library (for testing purpose).
+
+namespace onnxruntime {
+
+std::vector<std::string> GetStackTrace() { return {}; }
+
+namespace math {
+
+uint16_t floatToHalf(float f) { return 0; }
+
+}  // namespace math
+
+}  // namespace onnxruntime
+
+size_t MlasNchwcGetBlockSize() { return 0; }
```

Comment:
Interesting, the schemas depends on "MlasNchwcGetBlockSize"?  Later on probably we can refactor the code and make it better. In my opinion, "abort()" is a bit better than "return 0" in such cases.